admin_manual/configuration_server/occ_command.rst

```diff
@@ -62,13 +62,10 @@ Display your ownCloud version::
 Query your ownCloud server status::
  
  $ sudo -u www-data php occ status
-   Array
-   (
-    [installed] => true
-    [version] => 8.1.0.3
-    [versionstring] => 8.1.0
-    [edition] => 
-   )
+   - installed: true
```

Comment:
See https://github.com/owncloud/documentation/pull/1098#discussion_r29645848

Should the `--output=json(_pretty)` be explained here aswell? @carlaschroder